core/app/models/spree/product/scopes.rb

```diff
@@ -63,9 +63,10 @@ def self.property_conditions(property)
     #
     #   SELECT COUNT(*) ...
     add_search_scope :in_taxon do |taxon|
-      includes(:classifications).
-      where("spree_products_taxons.taxon_id" => taxon.self_and_descendants.pluck(:id)).
```

Comment:
If removing `pluck(:id)` is not related with the change I'd remove it from this PR, or at least from this commit. It could be misleading since one could think that removing `pluck` is (also) what fixed here.

Reply:
Yes, you are correct. I force-pushed the first commit to restore `pluck(:id)`, as it makes no difference and should be more performant since we're not fetching data we don't need.